packages/react-charting/src/components/VerticalBarChart/VerticalBarChartRTL.test.tsx

```diff
@@ -0,0 +1,447 @@
+
+    // Assert
+    expect(bar.getAttribute('width')).toEqual('100');
+  });
+  test('Should render the bars with the specified colors', async () => {
+    // Arrange
+    // colors mentioned in the data points itself
+    render(<VerticalBarChart data={chartPoints} />);
+    await new Promise(resolve => setTimeout(resolve));
+    const bars = screen.getAllByText((content, element) => element!.tagName.toLowerCase() === 'rect');
+
+    // Assert
+    expect(bars[0].getAttribute('fill')).toEqual('#0078d4');
+    expect(bars[1].getAttribute('fill')).toEqual('#002050');
+    expect(bars[2].getAttribute('fill')).toEqual('#00188f');
+  });
+  test('Should render the bars with the a single color', async () => {
+    // Arrange
+    render(<VerticalBarChart data={chartPoints} useSingleColor={true} />);
+    await new Promise(resolve => setTimeout(resolve));
+    const bars = screen.getAllByText((content, element) => element!.tagName.toLowerCase() === 'rect');
+
+    // Assert
+    expect(bars[0].getAttribute('fill')).toEqual('#00bcf2');
+    expect(bars[1].getAttribute('fill')).toEqual('#00bcf2');
+    expect(bars[2].getAttribute('fill')).toEqual('#00bcf2');
+  });
+  test('Should render the bars with labels hidden', async () => {
+    // Arrange
+    const { container } = render(<VerticalBarChart data={chartPoints} hideLabels={true} />);
+    await new Promise(resolve => setTimeout(resolve));
+    const getByClass = queryAllByAttribute.bind(null, 'class');
+
+    // Assert
+    expect(getByClass(container, /barLabel/i)).toHaveLength(0);
+  });
+});
+
+describe('Vertical bar chart - Subcomponent line', () => {
+  test('Should render line along with bars', () => {
+    // Arrange
+    render(<VerticalBarChart data={pointsWithLine} />);
+    const line = screen.getAllByText((content, element) => element!.tagName.toLowerCase() === 'path');
+
+    // Assert
+    expect(line).not.toHaveLength(0);
+  });
+  test('Should highlight the data points and render the corresponding callout', () => {
+    // Arrange
+    const { container } = render(<VerticalBarChart data={pointsWithLine} />);
+    const getById = queryAllByAttribute.bind(null, 'id');
+    const firstPointonLine = screen.getAllByText((content, element) => element!.tagName.toLowerCase() === 'circle')[0];
+    expect(firstPointonLine).toBeDefined();
+    fireEvent.mouseOver(firstPointonLine);
+    // Assert
+    expect(firstPointonLine.getAttribute('visibility')).toEqual('visibility');
+    expect(getById(container, /toolTipcallout/i)).toHaveLength(0);
+  });
+});
+
+describe('Vertical bar chart - Subcomponent Legends', () => {
+  test('Should not show any rendered legends when hideLegend is true', () => {
+    const { container } = render(<VerticalBarChart data={pointsWithLine} hideLegend={true} />);
+    const getByClass = queryAllByAttribute.bind(null, 'class');
+    expect(getByClass(container, /rect/i)).toHaveLength(0);
```

Comment:
isn't rect used for the bars of the chart as well

Reply:
For bars, the tag is rect (<rect>)  and for legends, the classname has 'rect'

Reply:
I see. Add this as a comment here.

Reply:
done